File: rest-service/src/main/java/org/veupathdb/service/multiblast/util/Address.java
package org.veupathdb.service.multiblast.util;

public class Address
{
  private static final String ProtocolDiv = "://";
  private static final String HttpProtocol = "http" + ProtocolDiv;

  public static String http(String value) {
    return value.indexOf(ProtocolDiv) > -1
      ? value
      : HttpProtocol + value;

  }
}

File: rest-service/src/main/java/mb/api/service/util/Address.kt
package mb.api.service.util

object Address
{
  private const val ProtocolDiv = "://"
  private const val HttpProtocol = "http$ProtocolDiv"

  fun http(value: String) = if (value.indexOf(ProtocolDiv) > -1) value else HttpProtocol + value
}
